cin_validator/rules/cin2022_23/rule_8736.py

```diff
@@ -0,0 +1,188 @@
+from typing import Mapping
+
+import pandas as pd
+
+from cin_validator.rule_engine import CINTable, RuleContext, rule_definition
+from cin_validator.test_engine import run_rule
+from cin_validator.utils import make_census_period
+
+# Get tables and columns of interest from the CINTable object defined in rule_engine/__api.py
+
+Assessments = CINTable.Assessments
+AssessmentAuthorisationDate = Assessments.AssessmentAuthorisationDate
+AssessmentActualStartDate = Assessments.AssessmentActualStartDate
+LAchildID = Assessments.LAchildID
```

Comment:
you'll need to import CINdetailsID here to be able to filter rows that don't have CINdetailsID in condition_1, this is because it's by assessment group, and we want to be sure we are only doing things per group as a child may have multiple